test/cpp/end2end/xds/xds_end2end_test_lib.cc

```diff
@@ -971,5 +1009,20 @@ XdsEnd2endTest::CreateTlsServerCredentials() {
   return grpc::experimental::TlsServerCredentials(options);
 }
 
+std::shared_ptr<ServerCredentials>
+XdsEnd2endTest::CreateMtlsSpiffeServerCredentials() {
+  std::string root_cert =
+      grpc_core::testing::GetFileContents(kSpiffeCaCertPath);
+  auto certificate_provider =
+      std::make_shared<grpc::experimental::StaticDataCertificateProvider>(
+          std::move(root_cert), MakeIdentityKeyCertPairForSpiffeMtlsCreds());
+  grpc::experimental::TlsServerCredentialsOptions options(
+      std::move(certificate_provider));
+  options.watch_root_certs();
+  options.watch_identity_key_cert_pairs();
+  options.set_cert_request_type(GRPC_SSL_REQUEST_CLIENT_CERTIFICATE_AND_VERIFY);
```

Comment:
This should probably be GRPC_SSL_REQUEST_AND_REQUIRE_CLIENT_CERTIFICATE_AND_VERIFY

Reply:
Done